### DISCUSSION TITLE
Predownload Images

### DISCUSSION THREAD

**Discussion by smooth-baking-powder** [nextcloud/all-in-one#4501] [Ideas] [answered]
Hi,

why we need to stop all Containers with the Mastercontainer and "update and start" Containers again? This Process needs a long Offlinetime. 

Why the Mastercontainer cant download the new Version before you have to stop and update the Container? For example the Ansible Playbook for Matrix do this. Before it stop and start the Containers, all images are downloaded and the Restartprocess is much faster.

Maybe you can add this for Nextcloud AIO.

Thanks

**Comment by szaimen** [comment]
This sounds like a good improvement. Thanks for the idea!

PRs are very welcome!

For anyone as help: this needs a new update check if the images that are available locally match the upstream ones. Then we can show a button to pull the images first.

**Comment by szaimen** [comment]
I've added this to the list in https://github.com/nextcloud/all-in-one/issues/5251 now

**Comment by szaimen** [comment, marked as answer]
This is now implemented for daily backups with v11.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
